rebasehelper/plugins/build_tools/rpm/copr_.py

```diff
@@ -41,6 +41,21 @@ class Copr(BuildToolBase):
     Class representing Copr build tool.
     """
 
+    OPTIONS = [
+        {
+            'name': ['--remove-copr-project-after'],
+            'default': 60,
+            'help': 'after how many days the created copr build will be '
+                    'deleted, -1 for never, defaults to %(default)s',
```

Comment:
I think a simple switch would be sufficient, something like `--copr-project-permanent`, defaulting to `False`.

Reply:
You are right that this is too complex. The number of days can be later changed in the web interface. So it should either be permanent or last 60 days?

Reply:
Exactly.